automol/util/vector.py

```diff
@@ -154,112 +152,101 @@ def unit_perpendicular(
     :type value_if_parallel: Vector
     :rtype: numpy.ndarray
     """
+    xyz1 = np.subtract(xyz1, orig_xyz)
+    xyz2 = np.subtract(xyz2, orig_xyz)
+    xyz3 = np.cross(xyz1, xyz2)
 
-    xyz1 = numpy.subtract(xyz1, orig_xyz)
-    xyz2 = numpy.subtract(xyz2, orig_xyz)
-    xyz3 = numpy.cross(xyz1, xyz2)
-
-    if numpy.linalg.norm(xyz3) > 1e-7:
+    if np.linalg.norm(xyz3) > 1e-7:
         uxyz3 = unit_norm(xyz3)
     else:
         uxyz3 = value_if_parallel
 
     return uxyz3
 
 
-def unit_bisector(xyz1, xyz2, orig_xyz, outer: bool = False) -> Vector:
+def unit_bisector(
+    xyz1: Sequence | np.ndarray,
+    xyz2: Sequence | np.ndarray,
+    orig_xyz: Sequence | np.ndarray,
+    outer: bool = False,
+) -> np.ndarray:
```

Comment:
Types look good -- nice work -- but let's put `Vector` instead of `np.ndarray` (even though they are the same).

Reply:
Also, we'll need to `require_vector_like` for the vector inputs.